src/ralph/data_center/tests/factories.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import factory
+from factory.django import DjangoModelFactory
+
+from ralph.data_center.models.physical import (
+    Accessory,
+    DataCenter,
+    DataCenterAsset,
+    Rack,
+    RackAccessory,
+    ServerRoom,
+)
+
+
+class DataCenterFactory(DjangoModelFactory):
+
+    name = factory.Sequence(lambda n: 'Data Center #{}'.format(n))
```

Comment:
use cycle to generate name https://github.com/allegro/ralph_assets/blob/94effeee0195b76ed25f1b1abfeed83765ebfb18/src/ralph_assets/tests/utils/assets.py 
https://github.com/allegro/ralph_assets/blob/94effeee0195b76ed25f1b1abfeed83765ebfb18/src/ralph_assets/tests/utils/assets.py#L62